bin/run_brick.rb

```diff
@@ -4,22 +4,22 @@
 require_relative '../lib/gooddata'
 
 DEFAULT_BRICK = 'hello_world_brick'
+BRICK_PARAM_PREFIX = 'BRICK_PARAM_'
 
-brick_type = ENV['BRICK_TYPE'] || DEFAULT_BRICK
+brick_type = ARGV.length > 0 ? ARGV[0] : DEFAULT_BRICK
```

Comment:
warning: Style/ZeroLengthPredicate: Use `!empty?` instead of `length > 0`. (rubocop)